lldb/source/Plugins/Language/Swift/FoundationValueTypes.cpp

```diff
@@ -30,14 +30,23 @@ using namespace lldb_private;
 using namespace lldb_private::formatters;
 using namespace lldb_private::formatters::swift;
 
+static constexpr bool show_unavailable_workaround = true;
```

Comment:
Should we include a radar/SR link?

Reply:
Usually we try not to put radar links into source code, because they are not that useful to open source contributors. (It's still there in the commit message so we can still find it with `git blame`. My plan is for this patch to never leave the 5.3 branch, so hopefully nobody will have to do more archeology on this.
Filing an SR is defintiely the _right_ thing to do here. But I don't intend to keep this issue open for very long :-)